layouts/millennium/r4/sidebar.html

```diff
@@ -73,6 +73,7 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
         <ul class="js-guides">
           <li><a href="/millennium/r4/financial/account/">Account</a></li>
           <li><a href="/millennium/r4/financial/coverage/">Coverage</a></li>
+          <li><a href="/millennium/r4/financial/charge-item/">ChargeItem</a></li>
```

Comment:
I think ChargeItem should be listed before coverage for alphabetic order.

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/64ddf564c76cd6938adb0bdff00feda5cff09c86#diff-db70aef4601cdc7697917f05be2110c0c9e05c4346f1e0c64a137e344b3d3bd6R76)

Reply:
Thanks!